File: bot/src/main/java/com/lm_pakkanen/radio_pele_java/models/message_embeds/StopEmbed.java
package com.lm_pakkanen.radio_pele_java.models.message_embeds;

import com.lm_pakkanen.radio_pele_java.interfaces.IEmbedBuilder;
import lombok.Getter;
import net.dv8tion.jda.api.EmbedBuilder;
import net.dv8tion.jda.api.entities.MessageEmbed;

@Getter
public final class StopEmbed implements IEmbedBuilder {

  private final MessageEmbed embed;

  public StopEmbed() {
    final EmbedBuilder embedBuilder = IEmbedBuilder.getEmbedBuilder();
    embedBuilder.setTitle("Hasta la Vista!");
    this.embed = embedBuilder.build();
  }
}

File: bot/src/main/kotlin/com/lm_pakkanen/radio_pele_java/models/message_embeds/StopEmbed.kt
package com.lm_pakkanen.radio_pele_java.models.message_embeds

import com.lm_pakkanen.radio_pele_java.interfaces.IEmbedBuilder
import net.dv8tion.jda.api.entities.MessageEmbed

class StopEmbed : IEmbedBuilder {
    override val embed: MessageEmbed

    init {
        val embedBuilder = IEmbedBuilder.getEmbedBuilder()
        embedBuilder.setTitle("Hasta la Vista!")
        this.embed = embedBuilder.build()
  }
}
